test/units/plugins/action/test_gather_facts.py

```diff
@@ -40,7 +41,7 @@ class TestNetworkFacts(unittest.TestCase):
     templar = Templar(loader=fake_loader)
 
     def setUp(self):
-        pass
+        C.config.initialize_base_defs()
```

Comment:
shouldn't be necessary if gather_facts illegal config mutation gets fixed, yeah?

Reply:
Removed it and updated the assertions for the tests